indexer-app/src/cmr/indexer/data/index_set_elasticsearch.clj

```diff
@@ -34,9 +35,8 @@
         ;; to start
         (when (= "dev" (cmr.common-app.config/release-version))
           (println "Here in create-index function creating" index-name)
-          ;; TODO: Generic work: print out anything with generic in the name
-          (when (= index-name "1_generic_grid") (println "mappings:" mapping))
-          (when (= index-name "1_generic_variable") (println "mappings:" mapping)))
+          ;;TODO: Generic work: remove this block when no longer needed.
+          (when (string/includes? index-name "1_generic")  (println "mappings:" mapping)))
```

Comment:
The `1_xxx` prefix as I understood it was for a concept that was never implemented. Could we name the index something more useful like `cmr_generic`?

Reply:
All indexer names start with 1_xxx. These are the current indexers that we have:
1_autocomplete
1_collections_v2
1_all_collection_revisions
1_deleted_granules
1_small_collections
1_services
1_all_service_revisions
1_subscriptions
1_all_subscription_revisions
1_tags
1_tools
1_all_tool_revisions
1_variables
1_all_variable_revisions
1_generic_grid
1_all_generic_grid_revisions
1_generic_variable
1_all_generic_variable_revisions
1_generic_dataqualitysummary
1_all_generic_dataqualitysummary_revisions
1_generic_orderoption
1_all_generic_orderoption_revisions
1_generic_serviceentry
1_all_generic_serviceentry_revisions
1_generic_serviceoption
1_all_generic_serviceoption_revisions

Reply:
my vote is for this as well. Actually I think it has been brought up before.